diff --git a/node_modules/expo-notifications/build/warnOfExpoGoPushUsage.js b/node_modules/expo-notifications/build/warnOfExpoGoPushUsage.js
index 8260405..cf50dda 100644
--- a/node_modules/expo-notifications/build/warnOfExpoGoPushUsage.js
+++ b/node_modules/expo-notifications/build/warnOfExpoGoPushUsage.js
@@ -4,10 +4,7 @@ let didWarn = false;
 export const warnOfExpoGoPushUsage = () => {
     if (isRunningInExpoGo() && !didWarn) {
         const message = `expo-notifications: Android Push notifications (remote notifications) functionality provided by expo-notifications was removed from Expo Go with the release of SDK 53. Use a development build instead of Expo Go. Learn more at https://docs.expo.dev/develop/development-builds/introduction/.`;
-        if (Platform.OS === 'android') {
-            throw new Error(message);
-        }
-        else if (__DEV__) {
+        if (!didWarn) {
             didWarn = true;
             console.warn(message);
         }
diff --git a/node_modules/expo-notifications/src/warnOfExpoGoPushUsage.ts b/node_modules/expo-notifications/src/warnOfExpoGoPushUsage.ts
index c6883b3..4b89adb 100644
--- a/node_modules/expo-notifications/src/warnOfExpoGoPushUsage.ts
+++ b/node_modules/expo-notifications/src/warnOfExpoGoPushUsage.ts
@@ -7,9 +7,7 @@ export const warnOfExpoGoPushUsage = () => {
   if (isRunningInExpoGo() && !didWarn) {
     const message = `expo-notifications: Android Push notifications (remote notifications) functionality provided by expo-notifications was removed from Expo Go with the release of SDK 53. Use a development build instead of Expo Go. Learn more at https://docs.expo.dev/develop/development-builds/introduction/.`;
 
-    if (Platform.OS === 'android') {
-      throw new Error(message);
-    } else if (__DEV__) {
+    if (!didWarn) {
       didWarn = true;
       console.warn(message);
     }
